### PR TITLE
Alpha experience platform extension - staging review

### PR DESCRIPTION
**Pull request by emdobrin**
PR description:
- Steps for configuring the Experience Platform mobile extension
- Description of the Experience Platform events
- Sending events using the mobile extension
- Server response handling (success and error scenarios)

### CODE REVIEW

File: alpha/experience-platform-extension/data-platform-setup.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Data Platform setup
+
+## Prepare Platform
+
+To be able to send data to [Adobe Data Platform](https://platform.adobe.com/) you will need to create an XDM schema and a dataset that uses that schema.
+
+- [Create a schema](https://www.adobe.io/apis/experienceplatform/home/tutorials/alltutorials.html#!api-specification/markdown/narrative/tutorials/schema_editor_tutorial/schema_editor_tutorial.md)
+- Assign the XDM ExperienceEvent class to your schema.
+- Add the following Mixins: 
+  - ExperienceEvent Application Details
```
[the remark this answers is not in the document]
Comment (rekrishn) [reply alpha/experience-platform-extension/data-platform-setup.md]: oK, that's fine then.

File: alpha/experience-platform-extension/experience-platform-events.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+The **Property** interface should be used when defining complex properties in your XDM Schema, for example any property that is no of primitive type, String or Date. 
+
+```java
+public interface Property {
+
+    /**
+     * Serialize this {@code Property} object to a map equivalent of its XDM schema.
+     * @return XDM formatted map of this {@code Property} object
+     */
```
[the remark this answers is not in the document]
Comment (rekrishn) [reply alpha/experience-platform-extension/experience-platform-events.md]: yep, that makes sense.

File: alpha/experience-platform-extension/experience-platform-events.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    /**
+     * Returns the identifier for this schema as defined in the Adobe Experience Platform.
+     * The identifier is a URI where this schema is defined.
+     * @return the URI identifier for this schema
+     */
+    String getSchemaIdentifier();
+
+    /**
+     * Serialize this {@code Schema} object to a map equivalent of its XDM schema.
+     * @return XDM formatted map of this {@code Schema} object
```
[the remark this answers is not in the document]
Comment (rekrishn) [reply alpha/experience-platform-extension/experience-platform-events.md]: ah I see! Can we say exactly what you said?
""* Serialize this {@code Schema} object to a map with the same format as its XDM schema.

[the remark this answers is not in the document]
Comment (rekrishn) [reply alpha/experience-platform-extension/experience-platform-events.md]: Thanks @emdobrin!

[the remark this answers is not in the document]
Comment (emdobrin) [reply alpha/experience-platform-extension/experience-platform-events.md]: Thanks @rekrishn , changed based on your suggestion for both Schema and Property.

File: alpha/experience-platform-extension/experience-platform-setup.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  - ExperienceEvent Application Details
+  - ExperienceEvent Environment Details
+  - ExperienceEvent Commerce Details
+- [Create a dataset](https://platform.adobe.com/dataset/overview) where your data should be sent by using the schema that you created earlier.
+
+## Requesting a configuration ID
+
+To use the Mobile SDK, you need a configuration ID, which ensures that your data is routed to the correct location. To obtain a configuration ID from your consultant or through Adobe Client Care, provide the following information:
+
+- Org ID: To locate this ID, complete the instructions in [Organizations and account linking](https://docs.adobe.com/content/help/en/core-services/interface/manage-users-and-products/organizations.html).
```
Comment (kevinlind) [inline alpha/experience-platform-extension/experience-platform-setup.md]: The formatting of this list appears a little cluttered. Maybe use dashes '-' to separate the config key with its description to provide more space between them?

- Org ID - To locate this ID, complete the instructions in [Organizations and account linking](https://docs.adobe.com/content/help/en/core-services/interface/manage-users-and-products/organizations.html).
- Dataset ID - Available in the dataset UI when you click on a dataset.
- Schema ID - Available at the end of the URL of the schema creation/viewing page.
- Friendly Name - The name that will be used in future UIs for this configuration.

Comment (emdobrin) [reply alpha/experience-platform-extension/experience-platform-setup.md]: Updated based on @rekrishn 's suggestions, will ask for her advice before changing them back.

File: alpha/experience-platform-extension/response-handling.md [hunk 5]
```diff
@@ -1,16 +1,26 @@
...
 
 Each event handle that is returned by the server is handled by the Experience Platform extension and is dispatched as a new mobile event to the extensions that are listening for it.
 
-If you integrated the Griffon extension in your mobile application, to display the events in your session, go to <https://griffon.adobe.com/> and in **Search Events**, type `Response event`.
+If you integrated the [Project Griffon extension](https://aep-sdks.gitbook.io/docs/beta/project-griffon) in your mobile application, to display the events in your session, go to <https://griffon.adobe.com/> and in **Search Events**, type `Response event`.
 
 ## Error handling
 
-If a server-side error occurs, the Experience Platform extension logs an error/warning message to logcat with the details. This message is dispatched as a new error response event for the mobile extensions to handle.
+If a server-side error occurs, the Experience Platform extension logs an error/warning message with the error or warning details, like in the following example:
```
Comment (rekrishn) [inline alpha/experience-platform-extension/response-handling.md]: Please make the last part of the sentence separate like this:
"If a server-side error occurs, the Experience Platform extension logs an error/warning message with the error or warning details. For example:"

Comment (emdobrin) [reply alpha/experience-platform-extension/response-handling.md]: @rekrishn implemented

**Pull request merged**